packages/flutter/lib/src/material/slider.dart

```diff
@@ -389,12 +390,17 @@ class _RenderSlider extends RenderBox {
       ..onTapDown = _handleTapDown
       ..onTapUp = _handleTapUp
       ..onTapCancel = _endInteraction;
-    _reaction = new CurvedAnimation(parent: state.reactionController, curve: Curves.fastOutSlowIn)
-      ..addListener(markNeedsPaint);
+    _reaction = new CurvedAnimation(
+      parent: state.reactionController,
+      curve: Curves.fastOutSlowIn,
+    )..addListener(markNeedsPaint);
     state.enableController.value = isInteractive ? 1.0 : 0.0;
-    _enableAnimation = new CurvedAnimation(parent: state.enableController, curve: Curves.easeInOut)
-      ..addListener(markNeedsPaint);
+    _enableAnimation = new CurvedAnimation(
+      parent: state.enableController,
+      curve: Curves.easeInOut,
+    )..addListener(markNeedsPaint);
     state.positionController.value = _value;
+    state.positionController.addListener(markNeedsPaint);
```

Comment:
as a general rule, render objects should listen when attached and unlisten when detached.

it's probably fine here because you have a 1:1 relationship with an object with a more finite lifecycle, but you should add a comment explaining why it's ok that you never remove the listener.

Reply:
Done.